config/development_pinot.yaml

```diff
@@ -7,7 +7,7 @@ persistence:
         cluster: pinot-test
         table: "cadence_visibility_pinot"
         migration:
-          enabled: false
+          enabled: true
```

Comment:
I suspect this is not a config you intended to change?

Reply:
Yeah, this is just for test, OSS user should not need this by default